README.md

```diff
@@ -46,6 +50,9 @@ zome_functions! {
 ### Specification for App Development
 As new features, or changes to the HDK (and the API) are being designed, use cases will be added to an example app and put as changes to a pull request to its [repository](https://github.com/holochain/app-spec-rust). The repository also integrates the feature set available in Holochain's main branch.
```

Comment:
I think its better to explicitely mention the app-spec-rust repo: "an example app in the [app-spec-rust repo](https://github.com/holochain/app-spec-rust)"